thirdparty/CMakeLists.txt

```diff
@@ -33,6 +35,12 @@ cmake_dependent_option(ADIOS2_USE_SYSTEM_ENET
   "NOT ADIOS2_USE_SYSTEM_EVPath" OFF
 )
 
+if(ADIOS2_USE_SYSTEM_JSON)
+  find_package(nlohmann_json 3.1.2 REQUIRED)
```

Comment:
might need an `NLohmannJson` CMake target alias here...

Reply:
fixed. the self-defined name `NLohmannJSON` differed from the official CMake target.
I updated the old CMake targets to stay consistent with the "upstream" name which avoids introducing aliases when using the library from an external location,

Reply:
- [ ] **Question**: how can we auto-update (`sed?`) the `3.1.2` min-version here? Can we include this in the `update.sh`?